report/renderer/report-ui-features.js

```diff
@@ -324,25 +324,12 @@ export class ReportUIFeatures {
   }
 
   /**
-   * Downloads a file (blob) using a[download].
-   * @param {Blob|File} blob The file to save.
+   * Wrapper is redefined and used by DevTools.
+   *
+   * @param {Blob|File} blob
    */
   _saveFile(blob) {
```

Comment:
We need to keep this one around because DevTools overrides it
https://source.chromium.org/chromium/chromium/src/+/main:third_party/devtools-frontend/src/front_end/panels/lighthouse/LighthouseReportRenderer.ts;l=183-197?q=reportUIFeatures&ss=chromium%2Fchromium%2Fsrc:third_party%2Fdevtools-frontend%2Fsrc%2F